content/en/docs/started/workstation/minikf-aws.md

```diff
@@ -0,0 +1,191 @@
+1. Click on the **Continue to Subscribe** button
+
+   <img src="/docs/images/minikf-aws/minikf-aws-page.png"
+       alt="Continue to Subscribe"
+       class="mt-3 mb-3 p-3 border border-info rounded">
+
+1. Click on the **Accept Terms** button
+
+   <img src="/docs/images/minikf-aws/minikf-aws-accept-terms.png"
+       alt="Accept Terms"
+       class="mt-3 mb-3 p-3 border border-info rounded">
+
+1. Wait for a few minutes and the click on **Continue to Configuration**
+
+   <img src="/docs/images/minikf-aws/minikf-aws-continue-configuration.png"
+       alt="Continue to Configuration"
+       class="mt-3 mb-3 p-3 border border-info rounded">
+
+1. Select a **Region**. We recommend you keep the default values for **Delivery
+   Method** and  **Software Version**. Then click **Continue to Launch**
+
+   <img src="/docs/images/minikf-aws/minikf-aws-configuration.png"
+       alt="Configure MiniKF"
+       class="mt-3 mb-3 p-3 border border-info rounded">
+
+1. Here you can review your configuration details. Don’t forget to view the
+   **Usage Instructions** and keep them handy, as you are going to need them
+   later.
+
+   <img src="/docs/images/minikf-aws/minikf-aws-configuration-details.png"
+       alt="Configuration Details"
+       class="mt-3 mb-3 p-3 border border-info rounded">
+
+1. You can also select how to launch MiniKF or change the EC2 instance type. We
+   recommend you leave the default value for **Choose Action**. We recommend
+   that you keep the default **EC2 Instance Type** (m5.2xlarge) or choose an
+   even more powerful instance. Choosing a VM with reduced specs may make it
+   impossible to train ML models.
+
+   <img src="/docs/images/minikf-aws/minikf-aws-instance-type.png"
+       alt="Select Intance Type"
+       class="mt-3 mb-3 p-3 border border-info rounded">
+
+1. Choose an existing **VPC** or create a new one. If you create a new one, make
+   sure you click on the **refresh** icon to update the contents of the list, so
+   that your new VPC appears in it.
+
+   <img src="/docs/images/minikf-aws/minikf-aws-security-settings-vpc.png"
+       alt="Select a VP or create a new one"
+       class="mt-3 mb-3 p-3 border border-info rounded">
+
+1. Choose an existing **Subnet** or create a new one. If you create a new one,
+   make sure you click on the **refresh** icon to update the contents of the
+   list, so that your new subnet appears in it.
+
+   <img src="/docs/images/minikf-aws/minikf-aws-security-settings-subnet.png"
+       alt="Select a Subnet or create a new one"
+       class="mt-3 mb-3 p-3 border border-info rounded">
+
+1. We strongly recommend to create a new security group. Otherwise you may not
+   be able to access your MiniKF. To do so, click on **Create New Based On
+   Seller Settings**.
+   
+   ---
+   **NOTE**
+   
+   If you need to use an existing security group, **make sure it covers
+   the ports mentioned in the usage instructions** (see at the top of this
+   page).
```

Comment:
```suggestion
   **Note**: If you need to use an existing security group, **make sure it covers
   the ports mentioned in the usage instructions** (see at the top of this
   page).
```

Reply:
I agree, will change. I just followed what other people to in their docs. But I think that your suggestion makes sense.